src/types/users/partial_user.ts

```diff
@@ -0,0 +1,5 @@
+import { User } from "./user.ts";
+
```

Comment:
Missing link

Reply:
link for?

Reply:
Any documentation link regarding the fields of the partial user object.

Reply:
In that case, you should just do `Partial<User>`

Reply:
to be exact there is this but this is only for the team member object
![image](https://user-images.githubusercontent.com/72305210/113180677-e7d92380-9248-11eb-8439-1d745a28ca04.png)

Reply:
In that case, we should just go with your solution :stuck_out_tongue_closed_eyes: